README.MD

```diff
@@ -0,0 +1,3 @@
+# Jokes:wq
+
+To run the app, execute ```docker compose up``` from the project root directory. Access **localhost:8080/jokes** and provide an optional query parameter for the number of jokes requested.
```

Comment:
just a small note, it's `docker-compose` AFAIR

Reply:
I think this is an alias of ```docker compose``` because when I execute ```docker-compose``` without any arguments, it prints out the usage of ```docker compose```.